main.py

```diff
@@ -29,7 +29,7 @@ async def on_thread_create(thread: discord.Thread):
         print("new post!")
         with open(SETTING_JSON,'r')as f:#設定用json読み込み
             settings=json.load(f)
-        send_channel=bot.get_channel(settings[SEND_CHANNEL_ID])
+        send_channel = await bot.fetch_channel(SEND_CHANNEL_ID)
```

Comment:
キャッシュがあればキャッシュを使うようにするなら以下のような感じだと思います

```suggestion
        send_channel = bot.get_channel(SEND_CHANNEL_ID)
        if send_channel is None:
            send_channel = await bot.fetch_channel(SEND_CHANNEL_ID)
```

Reply:
あー、なるほど！
ありがとうございます。

Reply:
send_channel が None になるのを再現させる方法がわからないので、どうやってテストするのがよいだろう…